odl/tomo/backends/astra_cuda.py

```diff
@@ -463,10 +464,10 @@ def astra_cuda_bp_scaling_factor(proj_space, reco_space, geometry):
             # missing factor src_radius ** 2 in the ASTRA BP with
             # density weighting.
             det_px_area = geometry.det_partition.cell_volume
-            scaling_factor *= (src_radius ** 2 * det_px_area ** 2 /
-                               reco_space.cell_volume ** 2)
-
-        # TODO: add case with new ASTRA release
+            scaling_factor *= (src_radius ** 2 * det_px_area ** 2)
+    else:
```

Comment:
Scary catch-all else statement. Is this supposed to catch all above something? In that case be explicit about that and lets have a else with an error message.

Reply:
Could we just write that explicitly instead? Just easier to read (and less likely to be broken by some weird mistake later on). After that we merge.

Reply:
Sure. I've added explicit comments about the version ranges to all four cases for consistency.

Reply:
IMO the catch-all `else:` doesn't really do any harm. If it were replaced with an `elif ... >1.9.9dev` and we somehow screwed up such that none of the cases applied, the function would still return a (most likely wrong) value.

The very cautious version would be the `elif ... >1.9.9dev` case, plus an `else:` case that raises a `RuntimeError`. But maybe normal reasoning is good enough here.